src/runtime-rs/Makefile

```diff
@@ -371,8 +371,8 @@ endef
 
 # Returns the name of the kernel file to use based on the provided KERNELTYPE.
 # $1 : KERNELTYPE (compressed or uncompressed)
-define MAKE_KERNEL_NAME
-$(if $(findstring uncompressed,$1),vmlinux.container,vmlinuz.container)
+define MAKE_KERNEL_NAME_DB
+$(if $(findstring uncompressed,$1),vmlinux-dragonball-experimental.container,vmlinuz-dragonball-experimental.container)
```

Comment:
Will this affect that the runtime-rs will support more hypervisor in the future? It seems that this assumes runtime-rs only works with dragonball (although it does now).

Reply:
This one has a `DB` postfix.
So I believe when CLH/QEMU/other hypervisors are supported, they could introduce xxxx_CLH/xxxx_QEMU

Reply:
What other VMMs will have to do is create a new `MAKE_KERNEL_NAME` for their own usage.  But that's something they'll have to do when they become supported (which is not the case now).

Reply:
> What other VMMs will have to do is create a new `MAKE_KERNEL_NAME` for their own usage. But that's something they'll have to do when they become supported (which is not the case now).

I see, then this may affect the work related to @jodh-intel and @pmores. But I think this is good.

Reply:
Thanks for the ping @Ji-Xinyou.

I think this change is fine, but if we need to add further logic, I'd be in favour of making the `Makefile` call a utility script to return the name rather than having the logic in the `Makefile`. That would be clearer and easier to maintain.